common/core/src/commonMain/kotlin/com/trafi/ui/component/internal/BadgeConstants.kt

```diff
@@ -0,0 +1,37 @@
+package com.trafi.ui.component.internal
+
+import com.trafi.ui.theme.internal.BasicFontWeight
+import com.trafi.ui.theme.internal.ColorPalette.Gray300
+import com.trafi.ui.theme.internal.CurrentTheme
+import com.trafi.ui.theme.internal.copy
+import com.trafi.ui.theme.internal.dp
+
+class BadgeConstants(theme: CurrentTheme) {
+    val defaultContentColor = theme.colorPalette.onPrimary
+    val disabledColor = Gray300
+
+    val textStyleMedium = theme.typographyScale.textM.copy(fontWeight = BasicFontWeight.Bold)
+    val textStyleSmall = theme.typographyScale.textS.copy(fontWeight = BasicFontWeight.Bold)
+
+    val minHeightMedium = 24.dp
+    val minHeightSmall = 16.dp
+
+    val cornerRadiusMedium = theme.cornerRadiusScale.mediumBadgeRadius
+    val cornerRadiusSmall = theme.cornerRadiusScale.smallBadgeRadius
+
+    val iconHeight = 20.dp
+    val iconWidth = 20.dp
+
+    val subBadgeIconHeight = 16.dp
+    val subBadgeIconWidth = 16.dp
+
+    val spacer = theme.spacingScale.badgeSpacer
+
+    val horizontalPaddingMedium = theme.spacingScale.horizontalPaddingMedium
+    val horizontalPaddingSmall = theme.spacingScale.horizontalPaddingSmall
+
+    val verticalPaddingMedium = theme.spacingScale.verticalPaddingMedium
+    val verticalPaddingSmall = theme.spacingScale.verticalPaddingSmall
+
+    val border = (1.5f).dp
```

Comment:
I'd suggest adding an `internal val Double.dp get() = Dp(toFloat())` property to our common code so this can be written as `1.5.dp`, would be less syntax for designers to worry about.

Reply:
Also, border what? `borderWidth` perhaps?